### ISSUE TITLE
Unknown opcode when trying to request speaker

### ISSUE THREAD

**Issue by SylveonDeko** [Discord-Net-Labs/Discord.Net-Labs#94]
```System.Exception: WebSocket connection was closed
 ---> Discord.Net.WebSocketClosedException: The server sent close 4001: "Unknown opcode."
   at Discord.Net.WebSockets.DefaultWebSocketClient.RunAsync(CancellationToken cancelToken)
   --- End of inner exception stack trace ---
   at Discord.ConnectionManager.<>c__DisplayClass29_0.<<StartAsync>b__0>d.MoveNext()
[02:13:48 WRN] | #0 | Gateway | Failed to start the connection: System.Exception: WebSocket connection was closed
 ---> Discord.Net.WebSocketClosedException: The server sent close 4001: "Unknown opcode."
   at Discord.Net.WebSockets.DefaultWebSocketClient.RunAsync(CancellationToken cancelToken)
   --- End of inner exception stack trace ---
   at Discord.ConnectionManager.<>c__DisplayClass29_0.<<StartAsync>b__0>d.MoveNext()
System.Exception: WebSocket connection was closed
 ---> Discord.Net.WebSocketClosedException: The server sent close 4001: "Unknown opcode."
   at Discord.Net.WebSockets.DefaultWebSocketClient.RunAsync(CancellationToken cancelToken)
   --- End of inner exception stack trace ---
   at Discord.ConnectionManager.<>c__DisplayClass29_0.<<StartAsync>b__0>d.MoveNext()
[02:13:48 WRN] | #0 | Gateway | Failed to start the connection: System.Exception: WebSocket connection was closed
 ---> Discord.Net.WebSocketClosedException: The server sent close 4001: "Unknown opcode."
   at Discord.Net.WebSockets.DefaultWebSocketClient.RunAsync(CancellationToken cancelToken)
   --- End of inner exception stack trace ---
   at Discord.ConnectionManager.<>c__DisplayClass29_0.<<StartAsync>b__0>d.MoveNext()
System.Exception: WebSocket connection was closed
 ---> Discord.Net.WebSocketClosedException: The server sent close 4001: "Unknown opcode."
   at Discord.Net.WebSockets.DefaultWebSocketClient.RunAsync(CancellationToken cancelToken)
   --- End of inner exception stack trace ---
   at Discord.ConnectionManager.<>c__DisplayClass29_0.<<StartAsync>b__0>d.MoveNext()
[02:13:49 WRN] | #0 | Gateway | Failed to resume previous session
[02:13:50 WRN] | #0 | Gateway | Serializer Error
```

**Comment by SylveonDeko** [comment]
just tried it out and BecomeSpeakerAsync, MoveToSpeakerAsync give the same errors

**Comment by SylveonDeko** [comment]
i now find out it cant join normal VCs either

**Comment by Rutherther** [comment]
Fix is in #96, but further discussion will bee needed as I don't know if it doesn't break something else.

**Comment by quinchs** [comment]
Fixed in #96
